### DISCUSSION TITLE
ollama integration

### DISCUSSION THREAD

**Discussion by millosh** [rksm/org-ai#105] [Ideas]
It would be great to get ollama integration (https://github.com/jmorganca/ollama/blob/main/docs/api.md).

I tried to navigate the code and make the addition myself, but I still haven't my passed Lisp parentheses anxiety...

**Comment by doctorguile** [comment]
I have been using azure openai chatgpt4 from the beginning since it is the best and the 2nd best model doesn't even come close.

Well now the landscape is changing. 
After some testing, at least for image / vision request, it seems like gemini can be better or easier to use (less complaint from the guardrail)

So I have the same questions, is org-ai planning to support other backends in the future?

Thanks
